app/models/thredded/messageboard.rb

```diff
@@ -27,6 +27,14 @@ def ensure_position
       self.position ||= (created_at || Time.zone.now).to_i
     end
 
+    after_save :set_autofollow, on: :create
```

Comment:
Setting global auto-follow preference to true will not update it for the existing messageboards with this approach.

How about we just check both the global and per-messageboards preferences when calculating the autofollowers in `AutofollowUsers`?